db/sql/src/protocol.rs

```diff
@@ -301,6 +301,29 @@ impl HoprDbProtocolOperations for HoprDb {
         })?)
     }
 
+    async fn find_surb(&self, matcher: SurbMatcher) -> Result<(HoprSenderId, HoprSurb)> {
```

Comment:
How are SURBs in the db cleaned up?

Reply:
The cache has a TTL, so by accessing the cache, older entries get cleaned up.